app/controllers/state_file/intake_logins_controller.rb

```diff
@@ -39,7 +39,13 @@ def update
       if @form.valid?
         sign_in_and_redirect
       else
-        @records.each(&:increment_failed_attempts)
+        if @records&.any?
```

Comment:
will `@records` be `nil`? probably not, but since this logic is complicated, not going to take anything for granted. `.present?` alone was not a good check, since i want to make sure you can loop over the collection